docs/scos/user/intro-to-spryker/contributing-to-documentation/style-formatting-general-rules.md

```diff
@@ -0,0 +1,218 @@
+
+2. In the `marketplace_dev_sidebar.yml` file, add URL for the *Glue API guides* element. You don’t have to write *index.html* in the end of the link, the link will work without it:  
+
+```
+- title: Glue API guides         
+  url: /docs/marketplace/dev/glue-api-guides/
+```
+
+3. Save the changes.
+
+If you now open the Glue API guides category, your page will look like this:
+
+![img](https://spryker.s3.eu-central-1.amazonaws.com/docs/scos/user/intro-to-spryker/contributing-to-documentation/landing-page-for-category.png)
+
+## Working with versions
+
+We have two types of versions: page versions and the global website version. Page versions are 
+
+The versions are managed in the *config.yml* file*.* 
+
+### Page version
+
+The *page version* is the version of the **current page**. Therefore, they apply to the [versioned pages](https://github.com/spryker/spryker-docs/tree/master/docs/scos/user/shop-user-guides) only. The page versions reside in the *version-related scopes* section of the *config.yml* file:
+
+![config yaml](https://spryker.s3.eu-central-1.amazonaws.com/docs/scos/user/intro-to-spryker/contributing-to-documentation/config-yaml.png)
+
+To add a new version for the versioned categories:
+
+1. In the *config.yml* file, *version-related scopes* section add the version with the *scope* and *values* parameters as shown in the picture above.
+2. In the main categories for which the version you added should exist, create the folder named as the *version* value from step one. For example, if  you created version `202221.0` in *config.yml* -> *version-related scopes,* the folder should also be called `202221.0`.
+3. Add the file that should be in the new version, to the folder:
+
+![files list](https://spryker.s3.eu-central-1.amazonaws.com/docs/scos/user/intro-to-spryker/contributing-to-documentation/files-list.png)
+
+You can make a reference to the currently opened version of the versioned page using the `{{page.version}}` variable in text and in links. For example, if you open a document in version `202108.0` in the editor and write there:
+
+```
+This feature requires version {{page.version}} of the Merchants feature.
+For details on the feature, see [Merchant feature overview](/docs/marketplace/user/features/{{page.version}}/merchants/merchants-feature-overview.html).
+```
+
+on the website, your text will look like this:
+
+```
+This feature requires version 202108.0 of the Merchants feature.
+For details on the feature, see Merchant feature overview(link to merchants-feature-overview.html version 202108.0).
+```
+
+{% info_block infoBox "Info" %}
+
+You can use the `{{page.version}}` variable only when you refer to versioned pages on versioned pages. To make a reference to the latest page version on a versioned page to an unversioned page, use the `{{site.version}}` variable described in the next section. To make a reference on an unversioned page to an unversioned page, do not use any variable.
+
```

Comment:
same here for both {{page.version}} and {{site.version}} variables. The main issue is present below as well, please check